azure-keyvault/azure/keyvault/key_vault_client.py

```diff
@@ -103,19 +107,21 @@ def create_key(
          deserialized response
         :param operation_config: :ref:`Operation configuration
          overrides<msrest:optionsforoperations>`.
-        :rtype: :class:`KeyBundle <azure.keyvault.models.KeyBundle>`
-        :rtype: :class:`ClientRawResponse<msrest.pipeline.ClientRawResponse>`
-         if raw=true
+        :return: :class:`KeyBundle <azure.keyvault.models.KeyBundle>` or
```

Comment:
FYI all these docstring update are new Autorest, improved by the feedback from docs.microsoft.com